kvbc/tools/object_store_utility/integrity_checker.cpp

```diff
@@ -75,15 +75,14 @@ std::pair<BlockId, Digest> IntegrityChecker::getLatestsCheckpointDescriptor() co
   LOG_INFO(logger_, "Last block: " << lastBlock);
   auto it = dynamic_cast<s3::Client*>(s3_dbset_.metadataDBClient.get())
                 ->getIterator<s3::Client::SortByModifiedDescIterator>();
-  auto [key, val] = it->seekAtLeast(Sliver::copy(checkpoints_prefix_.data(), checkpoints_prefix_.length()));
-  UNUSED(val);
+  const auto& kvdata = it->seekAtLeast(Sliver::copy(checkpoints_prefix_.data(), checkpoints_prefix_.length()));
```

Comment:
not sure why did you removed the parameter pack?
Why not:
`const auto &[key, _] = it->seekAtLeast(Sliver::copy(checkpoints_prefix_.data(), checkpoints_prefix_.length()));`

I use '_' for the value since it is unused. Both are const references, and this form is more redadble.

Reply:
gcc/g++ compilation fails for `[[maybe_unused]]`. Hence, not retaining parameter pack.